collections/soldiers.js

```diff
@@ -0,0 +1,36 @@
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+const configs = require('../configFile');
+
+// Connection URL
+const url = configs.dbUrl;
+
+// Database Name
+const dbName = configs.dbName;
+
+// Use connect method to connect to the server
+function newSoldier(db, data, callback) {
```

Comment:
try initializing this collection logic in a way that you won't have to send the non-changing db every time you call the method.